vue3-apollo-quasar/src/components/RepoAbout/Topics.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <div v-if="topics" class="wrapper">
+    <span v-for="topic in topics" :key="topic" class="topic">
+      {{ topic }}
+    </span>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+
+export default defineComponent({
+  name: 'Topics',
+  props: {
+    topics: Array,
+  },
+});
+</script>
+
+<style lang="scss" scoped>
+.wrapper {
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+}
+
+.topic {
+  background-color: rgb(221, 244, 255);
+  color: rgb(9, 105, 218);
+  margin: 0 0.375rem 0.333em 0;
+  cursor: pointer;
+  display: inline-block;
+  padding: 0 7px;
+  font-size: 0.75rem;
+  font-weight: 500;
+  white-space: nowrap;
+  border: 1px solid transparent;
+  border-radius: 2rem;
+  padding: 0 0.625rem 0 0.625rem;
```

Comment:
Please choose one version of padding to be used. We are declaring a general padding then overriding what we just declared.

## Sidenote
Most of this can also be abstracted out by using quasar classes